### PR TITLE
adding E3SM native grid test

### PR DESCRIPTION
**Pull request by davis278**
Adding an E3SM native grid test notebook to the scientific directory within Jupyter-notebooks.

### CODE REVIEW

File: scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.ipynb [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+    "python -i E3SM_ne30_Native_Grid_Test.py\n",
+    "```\n",
+    "The generic interactive Python command is:\n",
+    "```\n",
+    "python -i Name_of_Your_Notebook.py\n",
+    "```\n",
+    "\n",
+    "A separate window should open where you can adjust the sliders and click on the \"Configure\" button to access more features of the plot. Click and drag on the center of the plot to change the angle at which you are viewing the data.\n",
+    "\n",
+    "To exit the interactive mode, close the window and type control-D within your command line window fully exit the interactive mode and to get your command prompt back. (If for some reason control-D does not work, try control-Z.)\n",
```
Comment (downiec) [inline scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.ipynb]: To exit the interactive mode, close the window and type control-D within your command line window to fully exit the interactive mode and to get your command prompt back.

File: scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.ipynb [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+   "source": [
+    "# x.interact()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Save a 2D image of the 3D plot\n",
+    "When working inside a notebook, saving the above image to a separte .png file could be useful if you want to look at the image outside of the notebook. The line of code below will save the plot to a .png file in the same directory as this notebook. \n",
```
Comment (downiec) [inline scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.ipynb]: When working inside a notebook, saving the above image to a separate.png

File: scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+# If you want to interact with the 3D plot, save a python script from this notebook to run outside the notebook, then edit the python script to un-comment out the following line. Use a non-mesalib conda environment when running the Python script so you can interact with the 3D image in the window that opens after running the x.plot( v, dv3d ) command.
+
+# In[12]:
+
+
+x.interact()
+
+
+# ## Save a 2D image of the 3D plot
+# When working inside a notebook, saving the above image to a separte .png file could be useful if you want to look at the image outside of the notebook. The line of code below will save the plot to a .png file in the same directory as this notebook.
```
Comment (downiec) [inline scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.py]: When working inside a notebook, saving the above image to a separate .png file could be useful if you want to look at the image outside of the notebook.

File: scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.ipynb (hunk 1 above)
Comment (davis278) [reply scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.ipynb]: Thanks - fixed.

File: scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.py (hunk 3 above)
Comment (davis278) [reply scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.py]: Fixed.

File: scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.ipynb (hunk 2 above)
Comment (davis278) [reply scientific/E3SM_Native_Grid_Test/E3SM_ne30_Native_Grid_Test.ipynb]: Fixed.

**Pull request merged**